src/pyj/read_book/paged_mode.pyj

```diff
@@ -170,6 +164,7 @@ def current_page_width():
 def layout(is_single_page, on_resize):
     nonlocal _in_paged_mode, col_width, col_and_gap, screen_height, gap, screen_width, is_full_screen_layout, cols_per_screen, number_of_cols
     line_height(True)
+    scroll_viewport.initialize_on_layout()
```

Comment:
Note that this is not called in flow mode (yet) because flow mode actually doesn't care about the body's text direction, just the page progression direction from the spine.